Source/WebKit/WebProcess/Plugins/PDF/PDFPluginBase.mm

```diff
@@ -59,11 +60,201 @@
 #import <WebCore/ResourceResponse.h>
 #import <WebCore/ScrollAnimator.h>
 #import <WebCore/SharedBuffer.h>
+#import <WebCore/WebAccessibilityObjectWrapperMac.h>
 #import <wtf/cocoa/VectorCocoa.h>
 #import <wtf/text/TextStream.h>
-
 #import "PDFKitSoftLink.h"
 
+
+@interface WKAccessibilityPDFDocumentObject: NSObject {
```

Comment:
I wonder if we can stick this in its own file.
Also space before the :

Reply:
Will do

Reply:
Take too long to solve all the symbol problems for objective c. Might be a nice to have in the future.